src/wp-includes/script-loader.php

```diff
@@ -240,7 +240,11 @@ function wp_register_development_scripts( $scripts ) {
 	);
 
 	foreach ( $development_scripts as $script_name ) {
-		$assets = include ABSPATH . WPINC . '/assets/script-loader-' . $script_name . '.php';
+		$asset_path = ABSPATH . WPINC . '/assets/script-loader-' . $script_name . '.php';
+		if ( ! file_exists( $asset_path ) ) {
+			continue;
```

Comment:
Could we add a doing it wrong here?

Reply:
> Could we add a doing it wrong here?

It's not something that's been done wrong by the site owner if these files don't exist, the files are supposed to be created by the build process so there is nothing that can be done by the owner to fix it.